store/sql_upgrade.go

```diff
@@ -201,6 +201,13 @@ func UpgradeDatabaseToVersion35(sqlStore *SqlStore) {
 	// The rest of the migration from Filenames -> FileIds is done lazily in api.GetFileInfosForPost
 	sqlStore.CreateColumnIfNotExists("Posts", "FileIds", "varchar(150)", "varchar(150)", "[]")
 
+	// Increase maximum length of the Channel table Purpose column.
+	if utils.Cfg.SqlSettings.DriverName == model.DATABASE_DRIVER_POSTGRES {
```

Comment:
Also, I think this is resizing the column every time Mattermost starts up? This might be causing the errors that are preventing the server from starting.